server/src/data-layer/services/PricingService.ts

```diff
@@ -0,0 +1,18 @@
+import { UserAdditionalInfo } from "data-layer/models/firebase"
+import { MembershipType } from "./membershipType"
+
+export default class PricingService {
+  public async getMembershipType(
+    user: UserAdditionalInfo
+  ): Promise<MembershipType> {
+    if (user.university && user.returning) {
+      return MembershipType.UoaReturning
```

Comment:
For these we should also check if studentid is defined or not:
![image](https://github.com/UoaWDCC/uasc-web/assets/100653148/fd4fa707-03e5-49b6-bbc2-2664aa0292d4)